src/docfx.website.themes/default/styles/docfx.css

```diff
@@ -629,22 +628,16 @@ body .toc{
   font-size: 12px;
 }
 .affix > ul.level1 {
-  overflow: hidden;
+  width: 100%;
   padding-bottom: 10px;
   height: calc(100% - 100px);
   margin-right: -20px;
```

Comment:
Before I saw this pull request, I tried to hack around the problem by editing the CSS in a browser, and just removing this `margin-right: -20px;` setting seemed to do the trick. Is it needed for anything?

Reply:
I cannot think of why this is needed... It's ok to remove this line.

Reply:
I will try to work on this PR again today to address the earlier comments but specifically on this one, that should get you another 20px but will not eliminate the issue of truncating long lines if they go beyond that extra 20px.